flutter/shell/platform/tizen/flutter_tizen_texture_registrar.cc

```diff
@@ -45,12 +44,19 @@ int64_t FlutterTizenTextureRegistrar::RegisterTexture(
       return -1;
     }
   }
-  FlutterDesktopRendererType renderer_type = FlutterDesktopRendererType::kEGL;
-  if (dynamic_cast<TizenRendererEvasGL*>(engine_->renderer())) {
-    renderer_type = FlutterDesktopRendererType::kEvasGL;
+  std::unique_ptr<ExternalTexture> texture_gl = nullptr;
+
+#ifndef UNIT_TESTS
+  if (engine_->renderer()) {
+    texture_gl = engine_->renderer()->CreateExternalTexture(texture_info);
   }
-  std::unique_ptr<ExternalTexture> texture_gl =
-      CreateExternalTexture(texture_info, renderer_type);
+#else
+  texture_gl = std::make_unique<ExternalTextureSurfaceEGL>(
+      ExternalTextureExtensionType::kDmaBuffer,
```

Comment:
Just curios...
Is the test docker environment always `"EGL_EXT_image_dma_buf_import"`?
```
 else if (engine_->renderer() &&
                 engine_->renderer()->IsSupportedExtension(
                     "EGL_EXT_image_dma_buf_import")) {
        gl_extension = ExternalTextureExtensionType::kDmaBuffer;
      }
```